libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -365,23 +359,30 @@ bool AP_Proximity::get_horizontal_distances(Proximity_Distance_Array &prx_dist_a
     return drivers[primary_instance]->get_horizontal_distances(prx_dist_array);
 }
 
-// get boundary points around vehicle for use by avoidance
-//   returns nullptr and sets num_points to zero if no boundary can be returned
-const Vector2f* AP_Proximity::get_boundary_points(uint8_t instance, uint16_t& num_points) const
-{
-    if (!valid_instance(instance)) {
-        num_points = 0;
-        return nullptr;
+void AP_Proximity::get_num_layers_sectors(uint8_t& num_layers, uint8_t& num_sectors) const
```

Comment:
Ideally we shouldn't need to have this in AP_Proximity.  Instead it should be within the Boundary_3D class.